parsons/google/google_bigquery.py

```diff
@@ -794,7 +794,7 @@ def copy(
         quote: Optional[str] = None,
         schema: Optional[List[dict]] = None,
         max_timeout: int = 21600,
-        convert_dict_columns_to_json: bool = True,
```

Comment:
This is a breaking change because it renames a parameter. I don't know that it will impact many people because the parameter was only added four months ago - it might be that you're the only one using it - and if we are going to play fast and loose with the API I guess now's the time to do it. But alternatively you could just add a second parameter.

Reply:
All right I'm going to go ahead and merge this as is, hopefully it's not an issue for folks.